### ISSUE TITLE
Owner permissions constraint 

### ISSUE THREAD

**Issue by erikschlegel** [aztfmod/rover#115]
Why does rover have a hard requirement [check](https://github.com/aztfmod/rover/blob/62fc95233b31fc08818187e7ee5dd394fb7bf1b0/scripts/functions.sh#L192)  on owner level permissions for the target subscription? It’s unclear to me which azure operation warrants such an elevated role when CAF creates the initial launchpad LZ. I understand that you’ll need User Access Administrator permissions on the subscription in order for CAF to create role assignments. 

This has become a blocker for customers interested in adopting CAF due to security policy restrictions with granting owner level permissions.

In summary, would passing User Access Administrator to the check_subscription_required_role call suffice as a workaround?

**Comment by erikschlegel** [comment]
CC: @brockneedscoffee

**Comment by LaurentLesle** [comment]
@erikschlegel would the following or equivalent works for you
```bash
rover -lz xxxx -skip-permission-check
```

**Comment by erikschlegel** [comment]
Adding the `skip-permission-check` flag would work great. Thanks!

**Comment by xpillons** [comment]
I'm hitting the same with my partner. This is a blocking issue right now, workaround would be to update the rover locally and comment that line https://github.com/aztfmod/rover/blob/701a3bb522a354d25419d9546a8365a6e65075ba/scripts/tfstate_azurerm.sh#L6

**Comment by xpillons** [comment]
> @erikschlegel would the following or equivalent works for you
> 
> ```shell
> rover -lz xxxx -skip-permission-check
> ```

Can't you just retrieve the minimal right requirement from the connected user and check if they pass the test ?
